index.html

```diff
@@ -16,11 +16,16 @@
                   </a> 
                  
                 <div>
-                    <img src="images/Union.png" alt="enable-image">
+                    <img class="loose" src="images/Union.png" alt="enable-image">
                 </div>
+                <ul class="desktop-nav">
+                    <li><a href="#">portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
             </nav>
```

Comment:
- [ ] [OPTIONAL] It seems I forgot to mention this in the previous review 🤦‍♂️ but kindly make sure the navbar looks similar to the given [Figma design 🎨](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-848&t=07iRtZ9u5un02P85-0).
    - Your design:
      ![image](https://user-images.githubusercontent.com/104968236/229088507-0f4b5c6e-3047-4c97-8d9d-f650ee44663c.png)
    - Figma design:
      ![image](https://user-images.githubusercontent.com/104968236/229088669-d806cb78-0d87-49ff-970f-fd82aa0212b5.png)
- You can achieve this by simply adding `justify-content: space-around;` to the `.navbar` class for the desktop version 🖥.